CONTRIBUTING.rst

```diff
@@ -48,54 +58,43 @@ ones. Use the ``celeritas-gen.py`` script (in the ``scripts/dev`` directory) to
 generate skeletons for new files, and use existing source code as a guide for
 how to structure the decorations.
 
+.. _fork: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/about-forks
 .. _clang-format: https://clang.llvm.org/docs/ClangFormat.html
 
 
-Pull request process
---------------------
-
-Celeritas uses the "squash and merge" process to ensure continuity of the code
-history and provide easy bisecting because all commits pass all tests.
-Squashing eliminates the potential of broken commits and relieves developers of
-the burden of worrying about clean commit messages within a branch.
-
-Each pull request must be reviewed by at least one
-member of the :ref:`core team <roles>` who is knowledgeable about
-the section of code being modified. Physics code should be compared against
-reference papers and other codes such as Geant4. By the end of the code review,
-the reviewer should understand the code well enough to maintain it (by
-extension or modification) in the future. The review process must be based on
-*constructive feedback* ("here's a suggestion to make this better" or "did you
-consider what would happen if X?"), not *destructive feedback* ("this code is
-ugly").
+Submitting code changes
+-----------------------
 
-Reviews should not be started until the "draft" status has been removed (if it
-was set to begin with). Once a pull request is under review, *do not* rebase,
-squash, or otherwise alter the branch history. It causes GitHub to lose
-comments and causes notifications to redirect. Additionally it will lose
-information about whether the test passed and failed on previous commits.
+When you believe that you've made a nontrivial, self-contained improvement to
```

Comment:
I agree this could be interpreted as meaning trivial changes can be pushed directly, which I assume is not the intent? I would recommend being explicit with something like "All changes to the code go through pull requests"